redaxo/src/addons/phpmailer/boot.php

```diff
@@ -9,88 +9,12 @@
 
 $addon = rex_addon::get('phpmailer');
 
-if (!$addon->hasConfig('errormail')) {
-    $addon->setConfig('errormail', 0);
-}
-
-if (!$addon->hasConfig('security_mode')) {
-    $addon->setConfig('security_mode', true); // true = AutoTLS
-}
-
 if (!rex::isBackend() && 0 != $addon->getConfig('errormail')) {
     rex_extension::register('RESPONSE_SHUTDOWN', static function (rex_extension_point $ep) use ($addon) {
```

Comment:
```suggestion
    rex_extension::register('RESPONSE_SHUTDOWN', static function (rex_extension_point $ep) {
```

Das sollte die psalm-meldung beheben.